src/resources/ModifierTemplates/ModifierTemplateInterfaces.ts

```diff
@@ -0,0 +1,76 @@
+export interface ModifierModel {
+    /**
+     * Unique identifier
+     */
+    id: string;
+    /**
+     * English human friendly name
+     */
+    displayName: string;
+    /**
+     * Category the modifier belongs to.
+     * LICENSE and SYSTEM are more for internal use and might need to be filtered out from display
+     */
+    modifierType: ModifierType;
+    /**
+     * List of statements contained in the modifier
+     */
+    statements: StatementModel[];
+}
+
+export interface StatementModel {
+    /**
+     * The statement key to apply the value to
+     */
+    key: string;
+    /**
+     * The value to set on the key, always a string so "0" or "true" are correct
+     */
+    value: string;
+    /**
+     * How the value will be applied if there are other statements on the same key
+     */
+    operation: StatementOperation;
+}
+
+export interface StatementDetailModel {
```

Comment:
I would also rename this to ModifierStatementDetailModel
```suggestion
export interface ModifierStatementDetailModel {
```